### DISCUSSION TITLE
SR: 2 computer do not see each others.

### DISCUSSION THREAD

**Discussion by roobdesigns** [DistroAV/DistroAV#1019] [Q&A - Support]
### Operating System Version

WINDOWS 11 NDI DON'T CONNECT WITH OTHER LOCAL DESK 

### CPU

5700X

### GPU

3060

### NDI Devices

PC IN LOCAL NETWORK

### OBS Version

30.02

### OBS Installation Method

.EXE

### OBS-NDI Version

4.13

### OBS-NDI Installation Method

.EXE

### OBS Log [URL]

https://obsproject.com/logs/it2eROdfvcjsWwfp

### NDI Version

NDI SDK WIN64 03:20:51 FEB 17 2024 5.6.1.0

### Describe the bug

I installed all step by step, and i can't connect with the OBS NDI in my laptop and PC with Windows 10. I have windows 11 in my core desktop PC.  When all are installed and try connect to another OBS on my Local network the PC don't is show in the list, only the main PC.  And the Main pc is show in the Win10 pc but can see image.

**Comment by Trouffman** [comment]
Please check each of theses steps in the troubleshooting guide : 
https://github.com/obs-ndi/obs-ndi/wiki/2.-Troubleshooting

Make sure you have enabled the NDI output on your "sending" OBS.
Make sure your firewall is not blocking this. (see troubleshooting steps)

**Reply by FastFireFast** [reply]
First of all, thank you, I found the problem, my firewall on the relaying machine was the problem, but having it disabled is not an option, what can I do to solve it?

I wait your answer! 😉

**Reply by RayneYoruka** [reply]
Well... fix your firewall rules is the only solution... Allow the ports etc that NDI needs to work...

**Reply by paulpv** [reply]
https://docs.ndi.video/docs/white-paper/ndi-related-network-ports

**Reply by FastFireFast** [reply]
Thank you very much, that was what I needed! 🤗

**Reply by FastFireFast** [reply]
> Well... fix your firewall rules is the only solution... Allow the ports etc that NDI needs to work...

If I guessed, paulpv already gave me the ports it uses that I didn't know 😉
